src/sentry/replays/blueprints/api.md

```diff
@@ -238,6 +238,91 @@ Deletes a replay instance.
 
 - Response 204
 
+## Replay Accessibility Issues [/projects/<organization_slug>/<project_slug>/replays/<replay_id>/accessibility-issues]
+
+This resource does not accept any URI parameters and is not paginated. Responses are ingested whole.
+
+### Fetch Replay Accessibility Issues [GET]
+
+Retrieve a collection of accessibility issues.
+
+**Attributes**
+
+Issue Type:
+
+| Column    | Type                   | Description                                         |
+| --------- | ---------------------- | --------------------------------------------------- |
+| elements  | array[IssueElement]    | Array of elements matching the accessibility issue. |
+| help      | string                 | -                                                   |
+| help_url  | string                 | -                                                   |
+| id        | string                 | -                                                   |
+| impact    | Optional[enum[string]] | One of: 'minor', 'moderate', 'serious', 'critical'  |
+| timestamp | number                 | -                                                   |
+
+IssueElement Type:
+
+| Column       | Type                           | Description                                         |
+| ------------ | ------------------------------ | --------------------------------------------------- |
+| alternatives | array[IssueElementAlternative] | Array of solutions which could solve the problem.   |
+| element      | string                         | Array of elements matching the accessibility issue. |
+| target       | array[string]                  | Array of elements matching the accessibility issue. |
+
+IssueElementAlternative Type:
+
+| Column  | Type   | Description                           |
+| ------- | ------ | ------------------------------------- |
+| id      | string | String ID of the accessibility issue. |
+| message | string | Message explaining the problem.       |
+
+- Response 200
+
+  ```json
+  {
+    "meta": {
+      "total": 1
+    },
```

Comment:
lets flip this over to the `X-Hits` header because that seems to be the built-in way